Tests/OptimizelyTests-Common/DatafileHandlerTests.swift

```diff
@@ -191,15 +191,15 @@ class DatafileHandlerTests: XCTestCase {
             count += 1
             if count == 10 {
                 handler.stopPeriodicUpdates()
-                expection.fulfill()
                 seconds = Int(abs(now.timeIntervalSinceNow))
+                expection.fulfill()
             }
         }
         
         wait(for: [expection], timeout: 20)
         
         XCTAssert(count == 10)
-        XCTAssert(seconds == 10)
+        XCTAssert(seconds > 5 && seconds < 20)
```

Comment:
Is this change because of flakiness? if so, i think we should fix it in a way that no future updates are required to this.

Reply:
A good point. I like to have large enough range to avoid flakiness, but still want to keep it narrow enough for meaningful testing :)